### PR TITLE
feat: DEV-232 Update example app to use session management

### PR DESCRIPTION
**Pull request by Max-Stytch**
Updates the sample app to use session management.
- Session data is stored in local cookie
- Session data is checked on every request
- Logout

https://user-images.githubusercontent.com/89937743/133175851-43414abc-dcf0-49cb-b0b5-ae330de7ada2.mp4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ An example app using the Stytch Javascript SDK with HTML and Express.js
 2. Install dependencies: `npm install`
 3. Create your own .env file by running `cp .env_template .env`
 4. Fill in the placeholder values for `STYTCH_PROJECT_ID` and `STYTCH_SECRET` in your `.env` file. Get your Stytch API credentials from the API keys tab in your [Stytch dashboard](https://stytch.com/dashboard/api-keys).
-5. Fill in the placeholder value for `STYTCH_PUBLIC_TOKEN` in `public/signupOrLogin.html`. This can also be found on your [Stytch dashboard](https://stytch.com/dashboard).
+5. Fill in the placeholder value for `STYTCH_PUBLIC_TOKEN` in `public/signupOrLogin.html`. This can also be found on your [Stytch dashboard](https://stytch.com/dashboard/api-keys).
```
Comment (jeremy-stytch) [inline README.md:11]: I like that the public token is part of the `.env` setup now! I think these lines should be merged into one item.

**Review by jeremy-stytch — APPROVED** [review]
LGTM! 🎉

**Comment by Prince23will** [comment]
@linear-app[bot] can you help set up my account? 

_Sent from my TECNO CD7 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by chris-stytch** [comment]
@Max-Stytch mind doing a once over on the current state of this PR and merging if it looks good to you? I tested locally after updating to port 3000 and thinks look good to go.

**Comment by chris-stytch** [comment]
Closing pending new work on Sessions.

**Pull request closed without merging**